src/utils/test-utils.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC, ReactElement } from 'react';
+import { render, queries, RenderOptions } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+
+const AllTheProviders: FC<{ children: React.ReactNode }> = ({ children }) => {
+  return <BrowserRouter>{children}</BrowserRouter>;
+};
+
+const customRender = (
+  ui: ReactElement,
+  options?: Omit<RenderOptions, 'wrapper'>,
+) => render(ui, { wrapper: AllTheProviders, ...options });
+
+export * from '@testing-library/react';
+export { customRender as render };
```

Comment:
i understand why this is here, but I think you can eliminate this with the use of `BrowserRouter` in App.jsx instead of just `Routes`. See above suggestion for that.